[2020-03-27 14:30:20] skeletonlog.ERROR: Uncaught Exception ArgumentCountError: "Too few arguments to function Kptask\User\Service\UserService::__construct(), 4 passed in /vagrant/kptask/config/bootstrap.php on line 195 and exactly 5 expected" at /vagrant/kptask/packages/User/src/Service/UserService.php line 52 {"exception":"[object] (ArgumentCountError(code: 0): Too few arguments to function Kptask\\User\\Service\\UserService::__construct(), 4 passed in /vagrant/kptask/config/bootstrap.php on line 195 and exactly 5 expected at /vagrant/kptask/packages/User/src/Service/UserService.php:52)"} []
[2020-03-27 14:32:42] skeletonlog.ERROR: Uncaught Exception TypeError: "Argument 1 passed to Kptask\User\Validator\UserValidationManager::__construct() must be an instance of Kptask\User\Validator\Flash, instance of Tamtamchik\SimpleFlash\Flash given, called in /vagrant/kptask/config/bootstrap.php on line 180" at /vagrant/kptask/packages/User/src/Validator/UserValidationManager.php line 18 {"exception":"[object] (TypeError(code: 0): Argument 1 passed to Kptask\\User\\Validator\\UserValidationManager::__construct() must be an instance of Kptask\\User\\Validator\\Flash, instance of Tamtamchik\\SimpleFlash\\Flash given, called in /vagrant/kptask/config/bootstrap.php on line 180 at /vagrant/kptask/packages/User/src/Validator/UserValidationManager.php:18)"} []
[2020-03-27 14:33:25] skeletonlog.ERROR: Uncaught Exception Error: "Call to undefined method Kptask\Core\Validator\AbstractValidationManager::_construct()" at /vagrant/kptask/packages/User/src/Validator/UserValidationManager.php line 22 {"exception":"[object] (Error(code: 0): Call to undefined method Kptask\\Core\\Validator\\AbstractValidationManager::_construct() at /vagrant/kptask/packages/User/src/Validator/UserValidationManager.php:22)"} []
[2020-03-27 14:35:54] skeletonlog.ERROR: Uncaught Exception Error: "Class 'test606@net.com' not found" at /vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php line 29 {"exception":"[object] (Error(code: 0): Class 'test606@net.com' not found at /vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php:29)"} []
[2020-03-27 15:22:29] skeletonlog.NOTICE: E_NOTICE: Undefined variable: currentValidator {"code":8,"message":"Undefined variable: currentValidator","file":"/vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php","line":41} []
[2020-03-27 15:33:45] skeletonlog.ERROR: Uncaught Exception Error: "Call to undefined method SplStack::dequeue()" at /vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php line 40 {"exception":"[object] (Error(code: 0): Call to undefined method SplStack::dequeue() at /vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php:40)"} []
[2020-03-27 15:40:19] skeletonlog.ERROR: Uncaught Exception TypeError: "Argument 1 passed to Kptask\Core\Validator\AbstractValidator::succeedWith() must be an instance of Kptask\Core\Validator\AbstractValidator, null given, called in /vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php on line 42" at /vagrant/kptask/packages/Core/src/Validator/AbstractValidator.php line 22 {"exception":"[object] (TypeError(code: 0): Argument 1 passed to Kptask\\Core\\Validator\\AbstractValidator::succeedWith() must be an instance of Kptask\\Core\\Validator\\AbstractValidator, null given, called in /vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php on line 42 at /vagrant/kptask/packages/Core/src/Validator/AbstractValidator.php:22)"} []
[2020-03-27 15:49:50] skeletonlog.ERROR: Uncaught Exception TypeError: "Argument 1 passed to Kptask\Core\Validator\AbstractValidator::succeedWith() must be an instance of Kptask\Core\Validator\AbstractValidator, null given, called in /vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php on line 43" at /vagrant/kptask/packages/Core/src/Validator/AbstractValidator.php line 22 {"exception":"[object] (TypeError(code: 0): Argument 1 passed to Kptask\\Core\\Validator\\AbstractValidator::succeedWith() must be an instance of Kptask\\Core\\Validator\\AbstractValidator, null given, called in /vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php on line 43 at /vagrant/kptask/packages/Core/src/Validator/AbstractValidator.php:22)"} []
[2020-03-27 15:51:05] skeletonlog.ERROR: Uncaught Exception TypeError: "Argument 1 passed to Kptask\Core\Validator\AbstractValidator::succeedWith() must be an instance of Kptask\Core\Validator\AbstractValidator, null given, called in /vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php on line 42" at /vagrant/kptask/packages/Core/src/Validator/AbstractValidator.php line 22 {"exception":"[object] (TypeError(code: 0): Argument 1 passed to Kptask\\Core\\Validator\\AbstractValidator::succeedWith() must be an instance of Kptask\\Core\\Validator\\AbstractValidator, null given, called in /vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php on line 42 at /vagrant/kptask/packages/Core/src/Validator/AbstractValidator.php:22)"} []
[2020-03-27 15:51:32] skeletonlog.ERROR: Uncaught Exception TypeError: "Argument 1 passed to Kptask\Core\Validator\AbstractValidator::succeedWith() must be an instance of Kptask\Core\Validator\AbstractValidator, null given, called in /vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php on line 43" at /vagrant/kptask/packages/Core/src/Validator/AbstractValidator.php line 22 {"exception":"[object] (TypeError(code: 0): Argument 1 passed to Kptask\\Core\\Validator\\AbstractValidator::succeedWith() must be an instance of Kptask\\Core\\Validator\\AbstractValidator, null given, called in /vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php on line 43 at /vagrant/kptask/packages/Core/src/Validator/AbstractValidator.php:22)"} []
[2020-03-27 17:02:08] skeletonlog.NOTICE: E_NOTICE: Undefined variable: var_dump {"code":8,"message":"Undefined variable: var_dump","file":"/vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php","line":37} []
[2020-03-27 17:02:08] skeletonlog.ERROR: Uncaught Exception Error: "Function name must be a string" at /vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php line 37 {"exception":"[object] (Error(code: 0): Function name must be a string at /vagrant/kptask/packages/Core/src/Validator/AbstractValidationManager.php:37)"} []
[2020-03-27 18:18:35] skeletonlog.ERROR: Uncaught Exception Error: "Call to a member function fetchByEmail() on null" at /vagrant/kptask/packages/User/src/Validator/UserNormalizationValidator.php line 21 {"exception":"[object] (Error(code: 0): Call to a member function fetchByEmail() on null at /vagrant/kptask/packages/User/src/Validator/UserNormalizationValidator.php:21)"} []
[2020-03-27 20:12:10] skeletonlog.ERROR: Uncaught Exception TypeError: "Argument 1 passed to Kptask\User\Validator\EmailValidator::__construct() must implement interface Kptask\Core\Adapter\Email\EmailValidatorServiceInterface, null given, called in /vagrant/kptask/packages/User/src/Validator/UserValidationManager.php on line 71" at /vagrant/kptask/packages/User/src/Validator/EmailValidator.php line 26 {"exception":"[object] (TypeError(code: 0): Argument 1 passed to Kptask\\User\\Validator\\EmailValidator::__construct() must implement interface Kptask\\Core\\Adapter\\Email\\EmailValidatorServiceInterface, null given, called in /vagrant/kptask/packages/User/src/Validator/UserValidationManager.php on line 71 at /vagrant/kptask/packages/User/src/Validator/EmailValidator.php:26)"} []
[2020-03-27 20:33:01] skeletonlog.ERROR: Uncaught Exception Error: "Call to undefined method Kptask\User\Validator\EmailValidator::setRepository()" at /vagrant/kptask/packages/User/src/Validator/UserValidationManager.php line 49 {"exception":"[object] (Error(code: 0): Call to undefined method Kptask\\User\\Validator\\EmailValidator::setRepository() at /vagrant/kptask/packages/User/src/Validator/UserValidationManager.php:49)"} []
[2020-03-27 20:36:05] skeletonlog.ERROR: Uncaught Exception Error: "Call to undefined method Kptask\User\Validator\UserNormalizationValidator::getRepository()" at /vagrant/kptask/packages/User/src/Validator/UserNormalizationValidator.php line 36 {"exception":"[object] (Error(code: 0): Call to undefined method Kptask\\User\\Validator\\UserNormalizationValidator::getRepository() at /vagrant/kptask/packages/User/src/Validator/UserNormalizationValidator.php:36)"} []
